src/Networking/NexusMods.Networking.NexusWebApi/Services.cs

```diff
@@ -44,6 +45,10 @@ public static IServiceCollection AddNexusWebApi(this IServiceCollection collecti
         
         // Nexus API Key
         collection.AddAttributeCollection(typeof(ApiKey));
+
+        // Nexus GameDomain to GameID Mapping
+        collection.AddAttributeCollection(typeof(GameDomainToGameIdMapping));
```

Comment:
You can use the generated extension method instead: `collection.AddGameDomainToGameIdMappingAttributes` or whatever it's called.

Reply:
Might require some extra NuGet package, it wasn't available here.